setup.py

```diff
@@ -31,6 +31,7 @@
                    "pandas == 1.1.5; python_version <= '3.6'",
                    "pandas >= 1.1.5; python_version >= '3.7' and platform_python_implementation == 'CPython'",
                    "pandas == 1.2.5; platform_python_implementation == 'PyPy'",  # For pypy3.7, try again on pypy3.8
+                   "prometheus-client >= 0.10.0",
```

Comment:
Minor nit, can you move this up a few lines since it doesn't have a conditional.  The current requirements list is sorteds as "unconditional" requirements and "conditional" requirements.

Reply:
Done.